### PR TITLE
team page

### LINKED ISSUE #55: Create DeveloperCard component

### LINKED ISSUE #61: Create DeveloperList component

### PR DESCRIPTION
**Pull request by radvam**
Resolve #66  , resolve #61  , resolve #55

### CODE REVIEW

File: src/components/DeveloperCard/DeveloperCard.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.developer-card {
+  margin-top: 20px;
+  margin-bottom: 40px;
```
Comment (HyBeard) [inline src/components/DeveloperCard/DeveloperCard.scss]: + margin left/right

File: src/components/DeveloperList/DeveloperList.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.developer-list {
+  display: flex;
+  justify-content: space-between;
```
Comment (HyBeard) [inline src/components/DeveloperList/DeveloperList.scss]: **use**
justify-content: center;

**Review by HyBeard — COMMENTED** [review]
fix styles

File: src/components/DeveloperCard/DeveloperCard.scss (hunk 1 above)
Comment (radvam) [reply src/components/DeveloperCard/DeveloperCard.scss]: or margin: 20px 0 40px 0;

**Pull request merged**